Formula/l/llvm.rb

```diff
@@ -665,9 +668,10 @@ def caveats
         }
       EOS
 
+      rpath_flag = "-Wl,-rpath,#{lib}/#{Hardware::CPU.arch}-unknown-linux-gnu" if OS.linux?
       system bin/"clang", "-L#{lib}", "-fopenmp", "-nobuiltininc",
-                             "-I#{lib}/clang/#{llvm_version_major}/include",
-                             "omptest.c", "-o", "omptest"
+                          "-I#{lib}/clang/#{llvm_version_major}/include",
+                          rpath_flag, "omptest.c", "-o", "omptest"
```

Comment:
```suggestion
                          rpath_flag.to_s, "omptest.c", "-o", "omptest"
```
Oops, looks like you did need `#to_s`.